homeassistant/components/zwave_js/logbook.py

```diff
@@ -0,0 +1,139 @@
+"""Describe Z-Wave JS logbook events."""
+from __future__ import annotations
+
+from collections.abc import Callable
+
+from zwave_js_server.const import CommandClass
+
+from homeassistant.components.logbook.const import (
+    LOGBOOK_ENTRY_ENTITY_ID,
+    LOGBOOK_ENTRY_MESSAGE,
+    LOGBOOK_ENTRY_NAME,
+)
+from homeassistant.const import ATTR_DEVICE_ID, ATTR_ENTITY_ID
+from homeassistant.core import Event, HomeAssistant, callback
+import homeassistant.helpers.device_registry as dr
+
+from .const import (
+    ATTR_COMMAND_CLASS,
+    ATTR_COMMAND_CLASS_NAME,
+    ATTR_DATA_TYPE,
+    ATTR_DIRECTION,
+    ATTR_EVENT_LABEL,
+    ATTR_EVENT_TYPE,
+    ATTR_LABEL,
+    ATTR_VALUE,
+    DOMAIN,
+    ZWAVE_JS_NOTIFICATION_EVENT,
+    ZWAVE_JS_VALUE_NOTIFICATION_EVENT,
+    ZWAVE_JS_VALUE_UPDATED_EVENT,
+)
+
+
+@callback
+def async_describe_events(
+    hass: HomeAssistant,
+    async_describe_event: Callable[
+        [str, str, Callable[[Event], dict[str, str | int]]], None
+    ],
+) -> None:
+    """Describe logbook events."""
+    dev_reg = dr.async_get(hass)
+
+    @callback
+    def async_describe_zwave_js_notification_event(
+        event: Event,
+    ) -> dict[str, str | int]:
```

Comment:
Looks like it's always string values in the returned dict.
```suggestion
    ) -> dict[str, str]:
```